web/client/components/misc/AutocompleteWFSComboboxContainer.jsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2017, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+*/
+const React = require('react');
+const PropTypes = require('prop-types');
+const {AutocompleteWFSCombobox} = require('./AutocompleteWFSCombobox');
+const {createWFSFetchStream} = require('../../observables/autocomplete');
+
+class AutocompleteWFSComboboxContainer extends React.Component {
```

Comment:
remove this class the stream directly on rendering.

Reply:
i agree, i'll make the change.